css/styles.css

```diff
@@ -709,3 +710,458 @@ a {
   color: #172b4d;
   transform: 0.8s;
 }
+
+.nav-link .navigation-bar {
+  display: none;
+}
+
+/** Desktop Media css styles **/
+@media only screen and (min-width: 768px) {
+  .root-container {
+    display: grid;
+    all: unset;
+    width: 1440px;
+    height: 5125px;
+    background: #e5e5e5;
+  }
+
+  .header-section {
+    background-image: url("../image/header-desktop-bg.svg");
+    background-repeat: no-repeat;
+    flex-wrap: wrap;
+    align-items: center;
+    border-bottom-left-radius: 120px;
+    width: 1440px;
+    height: 916px;
+    left: 0;
+    top: 0;
+    margin: 0 auto 0 auto;
+  }

```

Comment:
- [ ] At `768px` width the content in the header section collapses to the left.  Kindly fix this issue.
-[ ] The Navigation links are too far from each other. Kindly reduce the space between them to make it look similar to the figma design. 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/179266291-eb904f6d-b8ac-41f1-bcfd-cdcb9bbcce4c.png) |  ![image](https://user-images.githubusercontent.com/60618877/179266165-bf970b42-b5e2-455f-bd41-e10a2fb7b06b.png)